demo/AppMode.ts

```diff
@@ -399,4 +347,20 @@ function getState() {
             return getDefaultState();
     }
 }
+
+function getState() {
+    const mode = Environment.searchParams.get('mode');
+    const flavorParam = Environment.searchParams.get('flavor');
+    const flavors = flavorParam ? flavorParam.split(';') : [];
+
+    let state = getModeState(mode);
+    flavors.forEach(flavor => {
+        const [key, valueString] = flavor.split('=');
+        const value = JSON.parse(valueString);
+
+        (state.tableProps as any)[key] = value;
+    });
```

Comment:
oh that's cool, so now I can tweak arbitrary props in the demo just by adding them to the url as `&flavor=...`? 🎉

Reply:
Yes. I've been using a "mode" param for a while but as more tests get added there's a lot of cases where we just want the same configuration + merging or + another small diff in the setup. Hence the "flavor" The syntax is a bit tricky at times because of the params encoding and since the prop's rhs must be json parseable.